src/components/course/ManageCoursePage.js

```diff
@@ -0,0 +1,119 @@
+import React, {PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import * as courseActions from '../../actions/courseActions';
+import CourseForm from './CourseForm';
+import toastr from 'toastr';
+import {browserHistory} from 'react-router';
+
+export class ManageCoursePage extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      course: Object.assign({}, this.props.course),
+      errors: {},
+      saving: false
+    };
+
+    this.saveCourse = this.saveCourse.bind(this);
```

Comment:
Can you move to use arrow functions and remove this `this.saveCourse.bind(this);`
This goes for all the `.bind(this)`